zipline-cli/src/main/resources-templates/zipline-cli-version.properties

```diff
@@ -0,0 +1 @@
+version=${projectVersion}
```

Comment:
If this is just for the version you can codegen a constant instead and save yourself a lot of build configuration and runtime code.